tox.ini

```diff
@@ -1,9 +1,11 @@
 # content of: tox.ini , put in same dir as setup.py
 [tox]
-envlist = py26,py27,py34
+envlist = py27,py34,py35
 
 [testenv]
+commands=nosetests
 deps=
    nose
    coverage
-commands=nosetests
+   -rrequirements.txt
```

Comment:
there is a test_requires that includes these libraries. Tox will not automatically run the setup.py. Instead of setup.py, I set it up to run requirements.txt to lock the versions.